label_studio/organizations/models.py

```diff
@@ -65,6 +66,8 @@ def soft_delete(self):
             ).first()
             self.user.save(update_fields=['active_organization'])
 
+        TaskLock.release_locks_for_user(self.user)
```

Comment:
could use the related names:
`self.user.task_locks.delete()`

Reply:
Yeah I like that better